python/lsst/daf/butler/registry/_registry.py

```diff
@@ -1385,27 +1388,27 @@ def queryDatasets(
 
         Returns
         -------
-        refs : `queries.DatasetQueryResults`
+        refs : `registry.queries.DatasetQueryResults`
             Dataset references matching the given query criteria.  Nested data
             IDs are guaranteed to include values for all implied dimensions
             (i.e. `DataCoordinate.hasFull` will return `True`), but will not
             include dimension records (`DataCoordinate.hasRecords` will be
-            `False`) unless `~queries.DatasetQueryResults.expanded` is called
-            on the result object (which returns a new one).
+            `False`) unless `~registry.queries.DatasetQueryResults.expanded` is
```

Comment:
```suggestion
            `False`) unless `~.queries.DatasetQueryResults.expanded` is
```

This seems to work for me without having to put the `registry` in the path. Similarly above on line 1391.